rewards/src/lib.rs

```diff
@@ -10,6 +10,7 @@ use orml_traits::RewardHandler;
 use scale_info::TypeInfo;
 use sp_core::U256;
 use sp_runtime::{
+	helpers_128bit::multiply_by_rational,
```

Comment:
use `multiply_by_rational_with_rounding` instead of `multiply_by_rational`.

Reply:
hm, seems it needs to upgrade polkadot deps. is not it? should than this PR merged after update OR can live with multiply_by_rational OR can make it work with u256?

Reply:
not sure, used u256 directly